k8s/namespaces/video-hearings/vh-user-api/vh-user-api.yaml

```diff
@@ -14,7 +18,3 @@ spec:
       replicas: 2
       useInterpodAntiAffinity: false
       image: sdshmctspublic.azurecr.io/vh/vh-user-api:latest
```

Comment:
is this right? it doesn't match our pattern of:

$register/$product/$component

project = team
component = app name

Reply:
project / team = vh
component / app = vh-user-api

Reply:
Unless i'm misunderstand it ?

Reply:
normally it would be:
`sdshmctspublic.azurecr.io/vh/user-api:latest`

Reply:
sorry I see what you mean, now